gap/miscellaneous.gd

```diff
@@ -1,7 +1,7 @@
 #############################################################################
 ##
 ##
-#W  miscelaneous.gd                       Manuel Delgado <mdelgado@fc.up.pt>
+#W  miscellaneous.gd                      Manuel Delgado <mdelgado@fc.up.pt>
```

Comment:
This is a bit weird, as that change should already be in `master` ?!? Perhaps you can rebase your branch?